src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,19 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, some street, #12-34, 272009";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses must be of format: BLOCK, STREETNAME, UNIT, POSTALCODE";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+[,].+[,].+[,].+";
+    private static final int BLOCK_INDEX = 0;
```

Comment:
2 things here:

- Good to see you have eliminated magic numbers in the code
- However, please take a look at the coding standards and find out what is the correct convention for naming constants